docs/sources/flow/tasks/send-profiles-to-pyroscope/index.md

```diff
@@ -0,0 +1,133 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/tasks/send-profiles-to-pyroscope/
+canonical: https://grafana.com/docs/agent/latest/flow/send-profiles-to-pyroscope/
+description: Send profiles to Pyroscope 
+title: Send profiles Pyroscope 
+weight: 120
+---
+
+# Send profiles to Pyroscope
+
+Learn how to configure {{< param "PRODUCT_NAME" >}} to collect [Pyroscope Server][] profiles and forward them to a [Pyroscope Server][].
+
+This topic describes how to:
+
+* Setup [Pyroscope Server][].
+* Setup {{< param "PRODUCT_NAME" >}} to collect profiles and send them to [Pyroscope Server][].
+
+## Components used in this topic
+
+* [pyroscope.write][]
+* [pyroscope.scrape][]
+
+## Before you begin
+
+* Download {{< param "PRODUCT_NAME" >}}.
+* Install [docker][] if it is not already installed.
+
+## Steps
+
+1. Setup Pyroscope
+     
+   The easiest path to setting up Pyroscope is using `docker run -it -p 4040:4040 grafana/pyroscope`. This will spin up a docker instance running Pyroscope on port `4040`. The server will autoscrape itself and after a few minutes data will appear. If you immedietly access the server it may error with `No applications available` until it has scraped itself.  
```

Comment:
```suggestion
   The easiest path to setting up Pyroscope is using `docker run -it -p 4040:4040 grafana/pyroscope`. This will spin up a Docker instance running Pyroscope on port `4040`. The server will autoscrape itself, and after a few minutes, data will appear. If you immediately access the server, it may error with `No applications available` until it has scraped itself.  
```